Milestone one

andresporras3423 opened the pull request:
- entity-relation model image in the doc folder
- 5 entities: user, friendship, post, like_post and comment
- user has many posts, friendships, like/dislike posts and comments
- friendship has the id of the user who create the friendship invitation and the id of the invited user
- friendship has a status attribute. If pending then is an invitation, if accepted then is a friendship, if rejected no invitation not friendship
- like_post save the user like (or dislike) to a post
- post contains the id of the user who created, also the content of the post   
- comment contains the id of the user and the id of the post

andresporras3423 committed:
first commit

andresporras3423 committed:
added ERD in docs folder, and edit README file

andresporras3423 committed:
edit README  file and add ERD in docs folder

andresporras3423 committed:
update README file

mosaaleb reviewed (changes requested):
### Well done @andresporras3423 @Salvador-ON :clap:

- [ ] User table has image_link column

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/imglink.png)


- [ ] **[OPTIONAL]** `friendships` table `status` can be boolean instead of `int` since it either true or false.

- [ ] **[OPTIONAL]** We don't need `like` boolean column in `like_post` table, if `[user_id, post_id] ` exist in this table, then the user likes a post, similarly otherwise.

Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

andresporras3423 committed:
update entity-relation diagram in doc folder

andresporras3423 commented:
* updated entity-relation diagram in doc folder
* added image_link in user entity
* changed type of status, in friendship entity
* like in post_like save true if user like a post, or save false if user dislike a post. When the user hasn't like or dislike a post then the record in the table doesn't exists.

fatymahmed reviewed (approved):
## Hi @andresporras3423  @Salvador-ON,
### Great job making the suggested changes🎉🎉
### It's a 👍🏾 from me!
## ✔️ Approved!

Good luck with your next projects.
 _Don't forget to submit your project completion form_
